frontend/public/components/storage-class-form.tsx

```diff
@@ -303,16 +305,16 @@ export class StorageClassForm_ extends React.Component<
       documentationLink: 'https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-file',
       parameters: {
         skuName: {
-          name: 'SKU Name',
-          hintText: 'Azure storage account SKU tier',
+          name: 'SKU Name', // t('storage-class-form~SKU Name')
+          hintText: 'Azure storage account SKU tier', // t('storage-class-form~Azure storage account SKU tier')
         },
         location: {
-          name: 'Location',
-          hintText: 'Azure storage account location',
+          name: 'Location', // t('storage-class-form~Location')
+          hintText: 'Azure storage account location', // t('storage-class-form~Azure storage account location')
         },
         storageAccount: {
-          name: 'Azure Storage Account Name',
-          hintText: 'Azure Storage Account Name',
+          name: 'Azure Storage Account Name', // t('storage-class-form~Azure Storage Account Name')
+          hintText: 'Azure Storage Account Name', // t('storage-class-form~Azure Storage Account Name')
```

Comment:
```suggestion
          hintText: 'Azure storage account name', // t('storage-class-form~Azure storage account name')
```

Reply:
Let's make the same case changes to the properties below. Too many to comment here :)

Reply:
done